peekingduck/pipeline/nodes/model/efficientdet.py

```diff
@@ -74,6 +75,10 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
         """
         # Currently prototyped to return just the bounding boxes
         # without the scores
-        bboxes, labels, scores = self.model.predict(inputs["img"])
+
+        # Convert BGR to RGB for EfficientDet model
```

Comment:
This comment can also be removed, because we practice [self documenting code](https://stackoverflow.com/questions/209015/what-is-self-documenting-code-and-can-it-replace-well-documented-code) and write comments only if good variable/function naming is not enough to suggest what the code does to the reader. In this case, it's pretty clear that this line converts BGR to RGB, so the comment can be left out.